config/settings.yml

```diff
@@ -1,6 +1,8 @@
 beta_banner:
   enabled: true
 
+google_site_verification: 'abyPMNSENI9-BVzLcKBsxfpsl_lOq-3hN588Ip12ZTk'
```

Comment:
Should this go in a public repo?

Reply:
It gets published in a meta tag and is publicly viewable in the page source so I don't think it should matter where we store it.